app/views/submissions/_form.html.erb

```diff
@@ -63,8 +67,26 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" style="display:none"><i class="fas fa-check"></i> <%=t('submission.correct_file_size')%></div>
```

Comment:
I guess I am throwing stones in the glass house here as I often neglect this in my own code, but it would be nice if these `display: none;` styles for the elements in the next four lines would be put in `submissions.scss` (which should then be imported in `application.scss`